course_catalog/serializers.py

```diff
@@ -332,6 +333,13 @@ def to_internal_value(self, data):
             {"name": topic_name}
             for topic_name in get_ocw_topics(data.get("course_collections"))
         ]
+
+        department_number = data.get("department_number")
+        if department_number in OCW_DEPARTMENT_CODE_TO_NAME.keys():
+            department = OCW_DEPARTMENT_CODE_TO_NAME[department_number]
```

Comment:
Maybe `department` could be saved as the department number instead,  then do this transformation to name in the search serializer?

`serializers.OCWSerializer`
```python
    department = data.get("department_number")
````

`models.Course`
```python
    @property
    def department_title(self):
        return OCW_DEPARTMENT_CODE_TO_NAME[self.department]
```

`search.serializers.ESCourseSerializer`
```python
 "department_title",
```